webapi/src/main/scala/org/knora/webapi/messages/util/rdf/JsonLDUtil.scala

```diff
@@ -610,10 +626,22 @@ case class JsonLDObject(value: Map[String, JsonLDValue]) extends JsonLDValue {
    * @tparam T the type of the validation function's return value.
    * @return the return value of the validation function, or `None` if the value was not present.
    */
-  @deprecated("use getIdIriFromObject(String) instead")
+  @deprecated("use getIdIriInObject(String) instead")
   def maybeIriInObject[T](key: String, validationFun: (String, => Nothing) => T): Option[T] =
     maybeObject(key).map(_.toIri(validationFun))
 
+  /**
+   * Gets the IRI of the object value of this JSON-LD at a specific key.
+   *
+   * @param key the key of the optional value.
+   * @return the IRI of the object value of this JSON-LD at a specific key.
+   *         Returns [[None]] if the key is not found.
+   *         Fails if the value is not a [[JsonLDObject]].
+   *         Fails if the [[JsonLDObject]] does not have an [[JsonLDKeywords.ID]].
+   */
+  def getIdIriInObject(key: String): IO[String, Option[String]] =
```

Comment:
Shouldn't this be called `getIriFromObject`?

Reply:
fixed in 7febee5f1f41e504bb169180ec167a5efac32965